sklearn/ensemble/_forest.py

```diff
@@ -1783,6 +1799,10 @@ class labels (multi-output problem).
     n_features_ : int
         The number of features when ``fit`` is performed.
 
+        .. deprecated:: 1.0
```

Comment:
1.0. -> 1.0

Reply:
?

Reply:
ups it was the line under but you already corrected